parts/common/components.json

```diff
@@ -80,7 +80,7 @@
       "windowsVersions": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/pause",
-          "latestVersion": "3.9"
+          "latestVersion": "3.9-hotfix-20230808"
```

Comment:
You may also need to check whether there are wrong values in the recent PRs.

Reply:
Yeah, we need to go through all the image versions and check they match what is in RP.